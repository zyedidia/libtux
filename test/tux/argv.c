#include <assert.h>
#include <stdio.h>
int main(int argc, char** argv) {
    printf("argc: %d\n", argc);
    for (int i = 0; i < argc; i++) {
        printf("argv[%d]: %s\n", i, argv[i]);
    }
    assert(argv[argc] == NULL);
    printf("argv[%d]: (nil)\n", argc);
    return 0;
}
