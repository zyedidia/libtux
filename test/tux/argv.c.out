argc: 4
argv[0]: ../build/test/tux_argv.c.lfi
argv[1]: one
argv[2]: two
argv[3]: three
argv[4]: (nil)
